plugins/cilium.yaml

```
@@ -0,0 +1,54 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: cilium
+spec:
+  version: v0.1.0
+  shortDescription: Easily interact with Cilium agents running on a Kubernetes cluster.
```

```suggestion
  shortDescription: Easily interact with Cilium agents
```
kubernetes is redundant and everything above ~50chars is usually truncated.

---

Did you mean to shorten the `shortDescription`? AFAICT, it's unchanged.
Feel free to reword it such that it makes most sense to you.

---

Sure I did, but I had forgotten. Thanks @corneliusweig!